api/types/filters/parse.go

```diff
@@ -230,6 +230,24 @@ func (filters Args) Validate(accepted map[string]bool) error {
 	return nil
 }
 
+// ValidateBoolValue ensures that the field filter are valid boolean value.
+// It returns an error as soon as it finds an invalid field value.
+func (filters Args) ValidateBoolValue(field string) (bool, error) {
```

Comment:
Wondering if we should standardize on something like https://golang.org/pkg/strconv/#ParseBool